examples/undocumented/python/distance_sparseeuclidean.py

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env python
 import shogun as sg
-traindat = '../data/fm_train_real.dat'
-testdat = '../data/fm_test_real.dat'
+traindat = '../data/fm_train_sparsereal.dat'
+testdat = '../data/fm_test_sparsereal.dat'
 
 parameter_list = [[traindat,testdat],[traindat,testdat]]
 
 def distance_sparseeuclidean (train_fname=traindat,test_fname=testdat):
-	from shogun import RealFeatures, SparseRealFeatures, CSVFile
+	from shogun import SparseRealFeatures
+	import shogun as sg
 
-	realfeat=RealFeatures(CSVFile(train_fname))
-	feats_train=SparseRealFeatures(realfeat)
-	realfeat=RealFeatures(CSVFile(test_fname))
-	feats_test=SparseRealFeatures(realfeat)
+	feats_train=SparseRealFeatures(sg.libsvm_file(train_fname))
```

Comment:
this is a temporary add so I can make the examples work with sparse files. We would later on just give a filename/file and the factory figures out the type automatically so that we just have `sg.file`